src/commands/deploy/utils.ts

```diff
@@ -716,3 +723,31 @@ export async function uploadEnvVarsFromFile(
         );
     }
 }
+
+// Upload the project code to S3 for in-browser editing
+export async function uploadUserCode(name: string, region: string, stage: string): Promise<void> {
+    const tmpFolderProject = await createTemporaryFolder();
+    debugLogger.debug(`Creating archive of the project in ${tmpFolderProject}`);
+    const promiseZip = zipDirectory(
+        process.cwd(),
+        path.join(tmpFolderProject, "projectCode.zip"),
+        false,
+        [
+            "**/node_modules/*",
+            "./node_modules/*",
+            "node_modules/*",
+            "**/node_modules",
+            "./node_modules",
+            "node_modules",
+            "node_modules/**",
+            "**/node_modules/**",
+        ],
+    );
+
+    await promiseZip;
+    const presignedUrlForProjectCode = await getPresignedURLForProjectCodePush(region, name, stage);
+    return uploadContentToS3(
```

Comment:
I think we should clean `tmpFolderProject` here with something like:
```
await deleteFolder(path.dirname(tmpFolderProject));
```

Reply:
All temporary folders are already deleted at process exit:

https://github.com/Genez-io/genezio/blob/a5e3ea5878b5a41563738926dc30ab8e350ed837/src/index.ts#L40

Reply:
My bad, disregard my comment